### DISCUSSION TITLE
Casa tre piani : Progetto

### DISCUSSION THREAD

**Discussion by Asky82** [indomus/forum#2868] [Nuove idee, proposte, progetti e guide]
Ciao a tutti,

spero che questa sia la sezione giusta.

Ho un dubbio riguardo la gestione di dispositivi smart home come lampadine/dispositivi HUE i quali necessitano di un HUB , in caso di una casa mutipiano.

Non mi è chiaro se un HUB sarebbe sufficiente oppure no. Ho letto che le lampadine fanno da ponte quindi se fosse cosi basterebbe riuscire a creare un sistema di dispositivi a catena in ogni piano. Corretto?

Grazie per il vostro aiuto :)

**Comment by indomus** [comment]
Lo accetti un consiglio franco? Leggiti prima [tutto questo](https://indomus.it/nozioni-di-base/).

**Comment by Asky82** [comment]
Ciao,grazie per l'info.
Avevo già letto quegli articoli ma non ho trovato info che mi aiutassero a capire come gestire una casa su 4 piani (Taverna>Sottotetto) e se quindi un bridge e sufficiente oppure no..a seconda dei marchi.

Ho chiesto direttamente  HUE e Tado e ho ricevuto info interessanti che magari vale la pena condividere.

HUE:
per ogni bridge max 50 lampadine e max 12 accessori (un pò pochi gli ultimi mi sembra)
ogni dispositivo funziona da bridge a patto di stare nei 10 mt di distanza

Tado:
Max 25 devices
distnaza dal gateway Max 15 metri
potenziale per coprire una casa da 2/3 piani

Un saluto

Tado
